content/about/about-content.md

```diff
@@ -2,26 +2,46 @@
+EDGI’s work has been widely acknowledged, leading to EDGI members testifying before Congress on declines in EPA enforcement, and hundreds of mentions in leading national and international media such as The New York Times, The Washington Post, Vice News, and CNN. For more about our work, read our [2019 Annual Report](https://envirodatagov.org/publication/edgi-annual-report-2019/) and [2018 Annual Report](https://envirodatagov.org/publication/edgi-annual-report-2018/). 
 
 ## Environmental Data Justice
-A lot of the work in EEW stems from EDGI’s work around “Environmental Data Justice,” or EDJ. EDJ promotes the production and use of environmental data to challenge structures of oppression like fascism, racism, heteropatriarchy, and classism. We believe that data should serve communities and not just corporate polluters. We also believe that communities should be able to have more resources for collecting and analyzing environmental data while retaining the power to decide what they can do with it. You can read more about EDJ and find some related readings in the <a href="https://docs.google.com/document/d/1O7ytnzXWFkluiYE4Pulo_mCHs9jdNpPm8hw83aLU2pg/edit?usp=sharing" target=_blank >EDJ Syllabus here</a>.
+A lot of the work in EEW stems from EDGI’s work around “Environmental Data Justice,” or EDJ. EDJ promotes the production and use of environmental data in ways that challenge structures of oppression like fascism, racism, heteropatriarchy, and classism. We believe that data should serve communities and not just corporate polluters. We also believe that communities should have more resources for collecting and analyzing environmental data while retaining the power to decide what they do with it. You can read more about EDJ and find some related readings in the [EDJ Syllabus here](https://drive.google.com/drive/folders/1F0N3NOxcZBmlDW57IyAnSTQ8XZtESEn1).
+
+## EEW’s Domains of Care 
+EEW is organized around domains of care, with each collaborator on the project supporting one or more dimensions that we collectively agree is vital to working toward EDGI’s [mission, vision, and values](https://envirodatagov.org/about/mission-vision-values/#:~:text=EDGI%20fosters%20the%20stewardship%20and,enable%20government%20and%20industry%20accountability.). In particular, we aim to build a culture of collaboration that enacts our values as well as processes that help “realize a world that creates and maintains healthy, just, bountiful, and beautiful environs in which people thrive.” 
+
```

Comment:
Delete extra return | *In Markdown, hitting enter multiple times won't give you more spaces, so adding them can give you a false impression that you'll have that space on the site*

Reply:
I'm not sure I understand what you mean here, should I get rid of the space after  " thrive." " ?